-meeting-template.md

```diff
@@ -1,4 +1,4 @@
-# YYYY-MM-DD Toronto Workers Co-op: All-Hands Meeting
+# YYYY-MM-DD Hypha Worker Co-op: All Hands Meeting
```

Comment:
We went back and forth on the hyphen and you are using it in the handbook... so, which one is your preference? :joy:

Reply:
heh, i think i'd migrated to your way in the most recent push :)

But I personally prefer hyphenating any compound modifiers: https://www.grammarly.com/blog/hyphen/

I thought I understood the rules, but maybe I don't! I would have guessed this was "Hyphen with Compound Modifiers: Two-Word Adjectives Before Nouns"